### PR TITLE
[12.0][IMP] fieldservice_stock

### PR DESCRIPTION
**Pull request by murtuzasaleh**
#392

### CODE REVIEW

**Comment by murtuzasaleh** [comment]
@max3903 @brian10048 Can you please review this PR.

**Comment by brian10048** [comment]
Is there a reason to add a new notebook tab when we already have the inventory tab?
In #392 @max3903 suggested making the `move_ids` field hidden if it is not set

**Comment by max3903** [comment]
@murtuzasaleh No need for a new page. As @brian10048 said, you can use the "Inventory" page.

The field should be invisible if not set.

File: fieldservice_stock/views/fsm_order.xml [hunk 1]
```diff
@@ -70,6 +70,7 @@
                             <field name="state"/>
                         </tree>
                     </field>
+                    <field name="move_ids" readonly="1" attrs="{'invisible': [('move_ids', '!=', False)]}"/>
```
Comment (max3903) [inline fieldservice_stock/views/fsm_order.xml]:
```suggestion
                    <field name="move_ids" readonly="1" attrs="{'invisible': [('move_ids', '=', False)]}"/>
```

**Review by max3903 — APPROVED** [review]

**Review by brian10048 — APPROVED** [review]

**Pull request merged**